modules/administration/pages/content-lifecycle.adoc

```diff
@@ -34,6 +34,25 @@ The [guimenu]``Name`` field only accepts lowercase letters, numbers, periods (``
 . Continue creating environments until you have all the  environments for your lifecycle completed.
 You can select the order of the environments in the lifecycle by selecting an environment in the [guimenu]``Insert before`` field when you create it.
 
+== Advanced: Filter types
+{productname} allows you to create various types of filters to control the content on project build. Below is the list of supported filters:
+
+- package filtering:
+  - by name,
+  - by name, epoch, version, release and architecture.
+- patch filtering:
+  - by advisory name,
+  - by synopsis,
+  - by date,
+  - by affected package.
```

Comment:
```suggestion
  - by affected package,
  - by type.
```

Reply:
there's no need for punctuation on list items, as they aren't complete sentences. Additionally, adoc uses * (and **) for bullets.

Reply:
ok, I'll update my PR